src/soundio/soundmanagerconfig.cpp

```diff
@@ -393,6 +393,21 @@ unsigned int SoundManagerConfig::getAudioBufferSizeIndex() const {
 // This reflects the configured value only. In case of JACK the
 // setting of the JACK server is used.
 unsigned int SoundManagerConfig::getFramesPerBuffer() const {
+    if (m_api == MIXXX_PORTAUDIO_JACK_STRING) {
+        // in case of jack we configure the frames/period
+        if (m_audioBufferSizeIndex == 7) {
+            return 4096;
+        } else if (m_audioBufferSizeIndex == 7) {
```

Comment:
```suggestion
        if (m_audioBufferSizeIndex == 7) {
            return 4096;
        } else if (m_audioBufferSizeIndex == 7) {
```
Both conditions are the same, this can't be correct.
But these numbers are not self explaining. Can you use an enumeration or similar here.

Reply:
Fixed